lighthouse-core/audits/dobetterweb/uses-http2.js

```diff
@@ -3,24 +3,29 @@
  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
+'use strict';
 
 /**
  * @fileoverview Audit a page to ensure that resource loaded over its own
  * origin are over the http/2 protocol.
  */
 
-'use strict';
+/** @typedef {import('../../lib/dependency-graph/simulator/simulator')} Simulator */
+/** @typedef {import('../../lib/dependency-graph/base-node.js').Node} Node */
 
-const URL = require('../../lib/url-shim.js');
 const Audit = require('../audit.js');
+const ThirdPartyAudit = require('../third-party-summary.js');
+const ByteEfficiencyAudit = require('../byte-efficiency/byte-efficiency-audit.js');
+const Interactive = require('../../computed/metrics/lantern-interactive.js');
+const NetworkRequest = require('../../lib/network-request.js');
 const NetworkRecords = require('../../computed/network-records.js');
+const LoadSimulator = require('../../computed/load-simulator.js');
+const PageDependencyGraph = require('../../computed/page-dependency-graph.js');
 const i18n = require('../../lib/i18n/i18n.js');
 
 const UIStrings = {
-  /** Title of a Lighthouse audit that provides detail on whether the webpage uses HTTP/2 for resources it requests over the network. This descriptive title is shown to users when the page uses HTTP/2 for its requests. */
-  title: 'Uses HTTP/2 for its own resources',
-  /** Title of a Lighthouse audit that provides detail on whether the webpage uses HTTP/2 for resources it requests over the network. This descriptive title is shown to users when the page does not use HTTP/2 for its requests. */
-  failureTitle: 'Does not use HTTP/2 for all of its resources',
+  /** Imperative title of a Lighthouse audit that tells the user to enable HTTP/2. This is displayed in a list of audit titles that Lighthouse generates. */
+  title: 'Leverage HTTP/2',
```

Comment:
I would prefer keeping this language simpler. `Should use HTTP/2`.

Reply:
Every single opportunity right now is framed as an action. `Use HTTP/2` then?